test/images/stub-device-plugin/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=alpine:3.6
```

Comment:
You don't have to create BASEIMAGE if your base image is `scratch`

Reply:
Thanks, updated to use BASEIMAGE in Dockerfile.